README.rst

```diff
@@ -151,6 +151,12 @@ a light weight dependency.
 ``dask.bag`` depends on ``toolz`` and ``dill``.
 
 
+Notebooks
+---------
+
+Check example notebooks using dask at https://notebooks.anaconda.org/dask/.
```

Comment:
Sounds good. Done.

Do you think the repo should have a folder with the notebooks or just leave them in the top repo directory for now?